bottles/backend/globals.py

```diff
@@ -52,6 +53,19 @@ def is_vkbasalt_available():
             return True
     return False
 
+def check_vrr_available():
+    """ Check if compositor supports VRR """
+    vrr_wayland_support = [
+                            "sway",
+                            "plasma",
+                          ]
+
+    desktop = environ.get("DESKTOP_SESSION")
+    check_wayland_session = True if DisplayUtils.display_server_type() == "wayland" else False
+    check_x11_session = True if DisplayUtils.display_server_type() == "x11" else False
+
+    return True if (desktop in vrr_wayland_support and check_wayland_session) or check_x11_session else False
+
```

Comment:
Let's cut this out, it's going to be unmanageable if this is how it's done in the future.